### PR TITLE
Fix and improvements

### PR DESCRIPTION
**Pull request by AntoninCurtit**
- Fixed a whitespace that was confusing during 2FA step.
```diff
- motdepasse: 123456
+ motdepasse:123456
```

---

- Improved FR syntax and fixed typo.

**Note :** Massive usage of uppercase in orginal document, this wasn't changed in FR but I consider doing so.

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -824,12 +824,12 @@ Basé sur le thème sombre Indigo"</string>
...
 
     <!-- Fuzzy -->
-    <string name="downloading_reddit_video_failed_cannot_get_cache_directory">"Échec du téléchargement: impossible d'accéder au répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_download_video">"Échec du téléchargement: impossible de télécharger la vidéo"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_video">"Échec du téléchargement: impossible d'enregistrer la vidéo dans le répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_audio">"Échec du téléchargement: impossible d'enregistrer l'audio dans le répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_mux">"Échec du téléchargement: impossible de multiplexer la vidéo et l'audio"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_mux_video">"Échec du téléchargement: impossible d'enregistrer la vidéo dans le répertoire public"</string>
+    <string name="downloading_reddit_video_failed_cannot_get_cache_directory">"Échec du téléchargement : impossible d'accéder au répertoire de cache"</string>
```
Comment (Docile-Alligator) [inline app/src/main/res/values-fr/strings.xml:827]: Hi! I am not familiar with French. But why is a space necessary here?

**Comment by Docile-Alligator** [comment]
Hi, I added a comment in the French translation. Could you look at it?

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (AntoninCurtit) [reply app/src/main/res/values-fr/strings.xml:827]: That's our punctation rules ;)
Like in English, you would write "What?", we would write " What ?". _(Well, « Quoi ? »...)_

**Comment by AntoninCurtit** [comment]
- [x] Seen and answered

**Comment by Docile-Alligator** [comment]
Thank you! That's nice and interesting.

**Comment by AntoninCurtit** [comment]
All thanks to you ! 🤜🤛

**Pull request merged**